tests/status/test_banner_mutations.py

```diff
@@ -73,15 +74,17 @@ def test_create_banner(db, clients, user_group, allowed):
                 "enabled": True,
                 "url": "https://www.example.com",
                 "urlLabel": "View more info here",
-                "startDate": datetime.datetime.now().isoformat(),
+                "startDate": timezone.now().isoformat(),
```

Comment:
This fixes the warning in the tests, but what if the user sends in a date time without a timezone? Will it still warn? Maybe we should attach a default timezone?

Reply:
Good point. It would still warn. I'm going to need to figure out all the points where the user sends in datetimes.

Reply:
This should be the only place as fair as I know. Usually datetimes are populated automatically.